engine/test_start_commands.py

```diff
@@ -0,0 +1,19 @@
+# coding = utf-8
+# Create date: 2018-11-05
+# Author :Hailong
+
+
+def test_start_commands(ros_kvm_with_paramiko, cloud_config_url):
+    command = 'ls /home/rancher | grep "test[0-5]"  | wc -l'
+    feed_back = '5'
+    client = ros_kvm_with_paramiko(cloud_config='{url}/test_start_commands.yml'.format(url=cloud_config_url))
```

Comment:
Need to reformat code.

Reply:
This result has been formatted and I think I don't need to modify it.